src/Simulation/Simulators/OpenSystemsSimulator/Decompositions/NativeInterface.qs

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// This namespace contains those intrinsics supported by the experimental
+// simulators native interface (NativeInterface.cs).
+
+namespace Microsoft.Quantum.Experimental.Native {
+
+    operation H(target : Qubit) : Unit {
```

Comment:
nitpick: Even though it is not strictly necessar, I'm personally in favor of including the `is Adj` in the signature even when the operation is explicitly declared as self-adjoint. It just makes it easier to see at a glance that the callable indeed supports the specialization.

Reply:
Good point, added.